app/routes/events/components/EventDetail/index.tsx

```diff
@@ -162,6 +164,8 @@ const EventDetail = () => {
       : selectPoolsForEvent(state, { eventId }),
   );
 
+  const showSkeleton = (fetching && isEmpty(event)) || !currentUser;
```

Comment:
Won't this prevent unauthenticated users from viewing event details?

Reply:
Ahh, right! Good catch! That makes fixing the type-errors a bit harder...

Reply:
I think this page will need to be refactored more later to fix all the types